Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob/Definitions/Result.cs

```diff
@@ -0,0 +1,12 @@
+﻿#pragma warning disable 1591
+namespace Frends.AzureBlobStorage.ReadBlob.Definitions
+{
+    public class Result
```

Comment:
You should add some XML-documentation about the Return-object, so the user knows what is returned from the task. This is important since the documentation is automatically generated from the source code.